### ISSUE TITLE
Create/update users with ORCID IDs from a spreadsheet

### ISSUE THREAD

**Issue by kelynch** [pulibrary/pdc_describe#127]
Using [this spreadsheet from PPPL](https://docs.google.com/spreadsheets/d/1DYt7lpmy6RMqXXDNXvsIPWn0Aerw390ceBYfQl61um4/edit#gid=0) as a resource, create/update users' accounts with first name, last name, and ORCID IDs.

We will need to add net IDs to this; I think there is a way to get them via the PPPL email address but I'm not sure.  I'll check and update this ticket.

**Comment by kelynch** [comment]
Net IDs added here - https://docs.google.com/spreadsheets/d/1U6AuWCLogVGBcNXmH4p6o8ZQc2nleyt0s0TedOpNkC0/edit#gid=0

**Comment by astrochun** [comment]
Following up on this matter, I do have about 70 more ORCIDs to include. Would it be best to edit the spreadsheet that you have? @kelynch, I think you have a way to easily retrieve the Princeton NetID, correct?

**Comment by kelynch** [comment]
Yes, [details here](https://github.com/pulibrary/pdc_describe/blob/main/docs/orcid_import.md).  I'll open a separate ticket for this and tag you.
